stripe/src/main/java/com/stripe/android/view/CardInputView.java

```diff
@@ -46,13 +48,13 @@
     private ExpiryDateEditText mExpiryDateEditText;
     private LockableHorizontalScrollView mScrollView;
     private View mCardNumberSpace;
+    private boolean mCardNumberIsViewed;
     private @ColorInt int mErrorColorInt;
+    private boolean mIsAmex;
+    private boolean mInitializedFlag;
     private int mScrollViewWidth;
     private int mScrollToPostion;
     private @ColorInt int mTintColorInt;
-    private boolean mCardNumberIsViewed;
-    private boolean mIsAmex = false;
```

Comment:
Are we relying on java defaulting the `boolean` to `false` rather than explicitly setting it to `false` as was done before?